app/src/main/res/navigation/nav_graph_categories.xml

```diff
@@ -36,7 +29,8 @@
     <fragment
         android:id="@+id/productDetailsFragment"
         android:name="com.example.internetshop.presentation.activity.fragments.ProductDetailsFragment"
-        android:label="ProductDetailsFragment" >
+        android:label='@string/label_product'
```

Comment:
Why do you use `label_product` which would be `Product #1` instead of product title?

Reply:
fixed